drivers/tests/TESTS/mbed_drivers/lp_ticker/main.cpp

```diff
@@ -143,8 +154,9 @@ void test_detach(void)
 {
     LowPowerTicker ticker;
     bool ret;
-    const float ticker_time_s = 0.1f;
-    const uint32_t wait_time_ms = 500;
+    const milliseconds ticker_time_s = 10ms;
```

Comment:
Is this correct? the previous value was 1/10th of a second by the looks of it, the new value is 1/100 th of a second ? Should it not be 100ms ?